### PR TITLE
Auth FE: Context

### LINKED ISSUE #15: Auth FE: Context
Save session data in local/session storage

### PR DESCRIPTION
**Pull request by DANIDIA**

### CODE REVIEW

File: src/contexts/SessionContext.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, { createContext, useState } from 'react';
+
+export const SessionContext = createContext({
+    /*eslint-disable-next-line*/
```
Comment (RomanovYurii) [inline src/contexts/SessionContext.js]: What eslint error do you get here?

File: src/contexts/SessionContext.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    setSessionID: (sessionID) => {},
+    sessionID: null,
+});
+
+export function SessionContextProvider({ children }) {
+    const [sessionID, setSessionID] = useState(null);
```
Comment (RomanovYurii) [inline src/contexts/SessionContext.js]: Don't be afraid of `undefined` - just handle it properly :)
```suggestion
    const [sessionID, setSessionID] = useState();
```

File: src/contexts/SessionContext.js (hunk 1 above)
Comment (DANIDIA) [reply src/contexts/SessionContext.js]: unused parameter `sessionId`

Comment (RomanovYurii) [reply src/contexts/SessionContext.js]: Try removing this parameter, it should work in JS afaik

**Review by RomanovYurii — APPROVED** [review]

**Pull request merged**